FirebaseSegmentation/Tests/Unit/SEGContentManagerTests.m

```diff
@@ -48,12 +57,22 @@ - (void)setUp {
   // Setup FIRApp.
   XCTAssertNoThrow([FIRApp configureWithOptions:[self FIRAppOptions]]);
   // TODO (mandard): Investigate replacing the partial mock with a class mock.
-  self.instanceIDMock = OCMPartialMock([FIRInstanceID instanceIDForTests]);
-  FIRInstanceIDResult *result = [[FIRInstanceIDResult alloc] init];
-  result.instanceID = @"test-instance-id";
-  result.token = @"test-instance-id-token";
-  OCMStub([self.instanceIDMock
-      instanceIDWithHandler:([OCMArg invokeBlockWithArgs:result, [NSNull null], nil])]);
+  //  self.instanceIDMock = OCMPartialMock([FIRInstanceID instanceIDForTests]);
+  //  FIRInstanceIDResult *result = [[FIRInstanceIDResult alloc] init];
+  //  result.instanceID = @"test-instance-id";
+  //  result.token = @"test-instance-id-token";
+  //  OCMStub([self.instanceIDMock
+  //      instanceIDWithHandler:([OCMArg invokeBlockWithArgs:result, [NSNull null], nil])]);
+
+  // Installations Mock
+  NSString *FID = @"fid-is-better-than-iid";
+  FIRInstallationsAuthTokenResult *FISToken =
+      [[FIRInstallationsAuthTokenResult alloc] initWithToken:@"fake-fis-token" expirationDate:nil];
+  self.installationsMock = OCMPartialMock([FIRInstallations installations]);
```

Comment:
I don't think a partial mock is needed here. I think we should replace it by a class mock:
```
self.installationsMock = OCMClassMock([FIRInstallations class]);
```

See an example of mocking FIRInstallations e.g. [here](https://cs.opensource.google/firebase-sdk/firebase-ios-sdk/+/master:Example/InstanceID/Tests/FIRInstanceIDTokenOperationsTest.m;l=409;drc=ef0f51e116cfe64d7cee1bc491cd09209a72218c). You'll need to stub `[FIRInstallations installationsWithApp:]` method instead `[FIRInstallations installation]` as you don't use a default instance.

As see []